src/abbrev.rs

```diff
@@ -6,6 +6,7 @@ use parser::{Error, Result, Format};
 use parser::{parse_unsigned_leb, parse_u8};
 use unit::UnitHeader;
 use std::collections::hash_map;
+use super::NamedDwarfSection;
```

Comment:
This could be just `use NamedDwarfSection;` unless there is some advantage to using a relative path here.